specs/v1.yaml

```diff
@@ -422,6 +422,441 @@ paths:
+      summary: Test Webhook
+      description: Execute a test on a webhook. This will send a test payload to the webhook endpoint
+      operationId: test-webhook
+      security:
+        - auth0:
+          - "read:webhooks"
+      parameters:
+        - name: id
+          in: path
+          description: the unique id of the webhook
+          schema:
+            type: string
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                eventType:
+                  type: string
+                  description: |-
+                    The event type to trigger
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Request-Id:
+              $ref: "#/components/headers/X-Request-Id"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Webhook"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks/{id}/messages:
+    get:
+      summary: List Webhook Messages
+      description: Lists the Webhook messages for a specific endpoint.
+      operationId: list-webhook-messages
+      security:
+        - auth0:
+          - "read:webhooks"
+      parameters:
+        - name: id
+          in: path
+          description: the unique id of the webhook
+          schema:
+            type: string
+          required: true
+        - name: pageLimit
+          in: query
+          description: the number of results to return; defaults to 10, max is 100
+          schema:
+            type: integer
+            default: 10
+        - name: pageToken
+          in: query
+          description: |-
+            A token to use to retrieve the next page of results. This is useful for paginating over many pages of results. If set, all other arguments are expected to be kept the same as previous calls.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: "#/components/responses/ListWebhookMessagesResponse"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks/{id}/messages/{messageId}:
+    get:
+      summary: Get Webhook Message
+      description: Gets a Webhook Message for a specific endpoint.
```

Comment:
Gets a message for a specific Webhook. Every message represents a unique HTTP request.. (let's elaborate on what messages are and explain them in the API docs)

Reply:
done